events/controller/controller.go

```diff
@@ -153,6 +153,8 @@ func (n *NatsController) Connect(ctx context.Context) error {
 	)
 	defer span.End()
 
+	startTS := time.Now()
```

Comment:
Consider trapping this as a metric so we can add to a dashboard.

Reply:
done